test/Benchmarks/src/Sieve/Sieve_Java_Script.enso

```diff
@@ -0,0 +1,119 @@
+from Standard.Base import Any, Meta, Integer, True, False, Nothing
+
+import project.Sieve.Sieve_Without_Types
+
+foreign js natural_in_js = """
+    class Gen {
+        constructor() {
+            this.n = 2
+        }
+
+        n() {
+            return this.n
+        }
+
+        next() {
+            this.n++
+            return this
+        }
+    }
+    return new Gen()
+
+compute_nth_prime_natural_in_js = Sieve_Without_Types.compute_nth_prime gen=natural_in_js
```

Comment:
These JavaScript benchmarks work with:
```
sbt:enso> runEngineDistribution --run test/Benchmarks Sieve
```
but not with `std-benchmarks/benchOnly Sieve`. Please help, @Akirathan!